README.md

```diff
@@ -1,24 +1,49 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-This project explores common infrastructure and coding standards for open source Google Cloud Platform projects.
+This project explores tools and practices 
+to provide consistency among Google Cloud Java libraries. The consistency
+minimizes the risk of dependency conflicts caused by the libraries.
 
-[Code Hygiene Dashboard](https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/dashboard/dashboard.html) (runs daily; work in progress)
+## Google Cloud Platform Java Dependency Dashboard
 
-To generate the dashboard from the root directory run:
+[Google Cloud Platform Java Dependency Dashboard](
+https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/dashboard/dashboard.html)
+(runs daily; work in progress) shows multiple checks on the consistency among
+Google Cloud Java libraries. For manually generating the dashboard, see
+[its README](./dashboard/README.md).
 
-```
-$ mvn clean install
-$ mvn exec:java -Dexec.arguments="-f boms/cloud-oss-bom/pom.xml"
-```
+## Java Library Best Practices
+
+[The Java Library Best Practices](./library-best-practices) are a set of rules
+that we believe will minimize problems for consumers of interconnected Java
+libraries.
+
+## Linkage Checker Enforcer Rule
+
+[Linkage Checker Enforcer Rule](./enforcer-rules) verifies that the transitive
+dependency tree of a Maven project or a [BOM](
+https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)
+does not have any [linkage errors](
```

Comment:
but it can have linkage errors

Reply:
Removed "a BOM", to avoid BOM member ordering issue.